.github/workflows/merge-main.yml

```diff
@@ -0,0 +1,14 @@
+name: Merge main
+on:
+  schedule:
+    - cron: "0 7 * * 1-5"  # At 07:00 UTC (00:00 PST, 03:00 EST), Monday through Friday
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure Gradle
+        uses: awslabs/aws-kotlin-repo-tools/.github/actions/merge-main@main
+        with:
+          exempt-branches: # Add any if required
```

Comment:
I was thinking this `exempt-branches` would be an input to the workflow. Each of our repositories can configure branches as needed rather than maintaining a global list